source/includes/fundamentals/code-snippets/timeSeries.go

```diff
@@ -33,21 +34,26 @@ func main() {
 
 	// begin create ts coll
 	db := client.Database("spring_weather")
+
+	// Creates a time series collection that stores "temperature" values over time
```

Comment:
I'm not sure if we should be using `timeseries` or `time series`, but it should be consistent

```suggestion
	// Creates a timeseries collection that stores "temperature" values over time
```